client/jetpack-cloud/sections/agency-dashboard/controller.tsx

```diff
@@ -49,8 +51,15 @@ export const agencyDashboardContext: Callback = ( context, next ) => {
 	}
 
 	const currentPage = parseInt( contextPage ) || 1;
+
+	const isDesiredUrl = context.path.includes( JETPACK_MANAGE_SITES_LINK );
+
 	context.header = <Header />;
-	context.secondary = <JetpackManageSidebar path={ context.path } />;
+	context.secondary = ! isDesiredUrl ? (
+		<JetpackManageSidebar path={ context.path } />
+	) : (
+		<SitesSidebar path={ context.path } />
+	);
```

Comment:
Maybe we could change this code by this. I think it should work:

```
	context.header = <Header />;
	context.secondary = showSitesDashboardV2 ? (
		<SitesSidebar path={ context.path } />
	) : (
		<JetpackManageSidebar path={ context.path } />
	);
```

Reply:
Works well, thanks! Added in b52212a0192a6a6c66655ac6bb82ecf54a9c4fb0